tests/ludwig/modules/test_recurrent_modules.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import torch
+
+from ludwig.modules import recurrent_modules
+
+logger = logging.getLogger(__name__)
+
+
+def test_recurrent_stack():
```

Comment:
You could potentially consolidate `test_recurrent_stack` and `test_recurrent_stack_NoSequenceSize` into a single test by using `pytest.mark.parameterize`.  E.g.
```
@pytest.mark.parameterize('sequence_size', [19, None])
```

This is probably good for now, but might be useful for future tests.